libsolidity/interface/CompilerStack.cpp

```diff
@@ -1482,7 +1482,19 @@ string CompilerStack::createMetadata(Contract const& _contract, bool _forIR) con
 {
 	Json::Value meta{Json::objectValue};
 	meta["version"] = 1;
-	meta["language"] = m_importedSources ? "SolidityAST" : "Solidity";
+	string sourceType;
+	switch (m_importedSourceType)
+	{
+	case ImportedSourceType::None:
```

Comment:
I guess this is technically in line with the previous implementation using `bool`, but it's kind of confusing that `None` returns `Solidity`.

Reply:
I actually think we should just call it `compilationSourceType` or something. The metadata also just calls it "sourceType". At some point we might also use "Yul" as source type, so this has nothing to do with it being imported or not.